subworkflows/nf-core/deepvariant/main.nf

```diff
@@ -0,0 +1,45 @@
+
+include { DEEPVARIANT_MAKEEXAMPLES        } from '../../../modules/nf-core/deepvariant/makeexamples/main'
+include { DEEPVARIANT_CALLVARIANTS        } from '../../../modules/nf-core/deepvariant/callvariants/main'
+include { DEEPVARIANT_POSTPROCESSVARIANTS } from '../../../modules/nf-core/deepvariant/postprocessvariants/main'
+
+workflow DEEPVARIANT {
+    take:
+    ch_input            // channel: [ val(meta), path(input), path(index), path(intervals)]
+    ch_fasta            // channel: [ val(meta2), path(fasta) ]
+    ch_fai              // channel: [ val(meta3), path(fail) ]
+    ch_gzi              // channel: [ val(meta4), path(gzi) ]
+    ch_model_type       // channel: val("wgs" | "wes")
```

Comment:
`ch_model_type       // channel: val("wgs" | "wes" | "ont_r104" | "pacbio" | "hybrid_pacbio_illumina" )`

TODO

Reply:
I end up removing this, as the model will be specified using the config